test/run-drun/gc-trigger.mo

```diff
@@ -5,7 +5,7 @@ import Prim "mo:prim";
 actor {
     let retained = Prim.Array_init<Nat>(6 * 1024 * 1024, 0);
     // GC is triggered during initialization
-    
+
```

Comment:
I believe the only adjustment to make this test pass again is to add an extra GC trigger call: https://github.com/dfinity/motoko/pull/4305

Reply:
Ok, not sure I understand why though - because I removed the second GC call?

Reply:
Yes, this was because the original GC trigger implementation had two GC calls.